miio/viomivacuum.py

```diff
@@ -268,124 +382,400 @@ def has_map(self) -> bool:
+
     @command(
         default_output=format_output(
-            "",
+            "\n",
+            "General\n"
+            "=======\n\n"
+            "Hardware version: {result.hw_info}\n"
             "State: {result.state}\n"
-            "Mode: {result.mode}\n"
-            "Error: {result.error}\n"
+            "Working: {result.is_on}\n"
+            "Battery status: {result.error}\n"
             "Battery: {result.battery}\n"
-            "Fan speed: {result.fanspeed}\n"
+            "Charging: {result.charging}\n"
             "Box type: {result.bin_type}\n"
-            "Mop type: {result.mop_type}\n"
-            "Clean time: {result.clean_time}\n"
-            "Clean area: {result.clean_area}\n"
+            "Fan speed: {result.fanspeed}\n"
             "Water grade: {result.water_grade}\n"
+            "Mop mode: {result.mop_mode}\n"
+            "Mop installed: {result.mop_installed}\n"
+            "Vacuum along the edges: {result.edge_state}\n"
+            "Mop route pattern: {result.mop_route}\n"
+            "Secondary Cleanup: {result.repeat_cleaning}\n"
+            "Voice state: {result.voice_state}\n"
+            "Clean time: {result.clean_time}\n"
+            "Clean area: {result.clean_area} m²\n"
+            "\n"
+            "Map\n"
+            "===\n\n"
+            "Current map ID: {result.current_map_id}\n"
             "Remember map: {result.remember_map}\n"
             "Has map: {result.has_map}\n"
             "Has new map: {result.has_new_map}\n"
-            "Mop mode: {result.mop_mode}\n",
+            "Number of maps: {result.map_number}\n"
+            "\n"
+            "Unknown properties\n"
+            "=================\n\n"
+            "Light state: {result.light_state}\n"
+            # "Order time: {result.order_time}\n"
+            # "Start time: {result.start_time}\n"
+            # "water_percent: {result.water_percent}\n"
+            # "zone_data: {result.zone_data}\n",
         )
     )
     def status(self) -> ViomiVacuumStatus:
         """Retrieve properties."""
         properties = [
-            "run_state",
-            "mode",
-            "err_state",
             "battary_life",
             "box_type",
+            "cur_mapid",
+            "err_state",
+            "has_map",
+            "has_newmap",
+            "hw_info",
+            "is_charge",
+            "is_mop",
+            "is_work",
+            "light_state",
+            "map_num",
+            "mode",
+            "mop_route",
             "mop_type",
-            "s_time",
+            # "order_time",
+            "remember_map",
+            "repeat_state",
+            "run_state",
             "s_area",
+            "s_time",
+            # "start_time",
             "suction_grade",
+            "v_state",
             "water_grade",
-            "remember_map",
-            "has_map",
-            "is_mop",
-            "has_newmap",
+            # "water_percent",
+            # "zone_data",
```

Comment:
Should these be below the "The following list of properties .." comment?

Reply:
done